rotkehlchen/exchanges/binance.py

```diff
@@ -1519,7 +1519,7 @@ def _api_query_list_within_time_delta(
         # Create required time references in milliseconds
         start_ts = Timestamp(start_ts * 1000)
         end_ts = Timestamp(end_ts * 1000)
-        offset = time_delta * 1000 - 1  # less than time_delta
+        offset = time_delta * 1000  # less than time_delta
```

Comment:
```suggestion
        offset = time_delta * 1000
```
Its not actually less than the time_delta anymore, so this comment should be removed, and also the docstring on this function needs adjusted as well.